config/config.exs

```diff
@@ -0,0 +1,10 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+use Mix.Config
+
+
+config :ex_limiter, :storage, ExLimiter.Storage.Memcache
```

Comment:
I wonder if we provide a local implementation as well so that you don't need to have memcache to run tests

Reply:
We'll need to do a bunch of ets garbage, which I really don't want to do now.  I'll look into it later if we need it.